metrics/sinks/stackdriver/stackdriver.go

```diff
@@ -45,6 +48,8 @@ type metricMetadata struct {
 }
 
 var (
+	// metrics metadata
```

Comment:
`Known metrics metadata` to be consistent with the capital letter + explain what metrics we're talking about